### PR TITLE
Reconcile `v0.3.5` with changes in branch `0.3.4`

### PR DESCRIPTION
**Pull request by d33bs**
After merging #134 I found that I had erroneously assumed the releases were tagged from `master`. On looking at the branch `0.3.4` I found there were differences that could be important to keep within the history of `master`. As a result, I merged the changes from `master` and `0.3.4` together, attempting to make decisions about conflicts where necessary. The only additional change added which is not from branch `0.3.4` is a version bump in `setup.cfg`.

All tests appear to pass using the same steps as mentioned in #134.

### CODE REVIEW

File: cytominer_database/ingest.py [hunk 1]
```diff
@@ -153,13 +157,8 @@ def seed(source, target, config_path, skip_image_prefix=True):
...
         # ingest the image CSV
         try:
-            into(
-                input=image,
-                output=target,
-                name=name.capitalize(),
-                identifier=identifier,
-                skip_table_prefix=skip_image_prefix,
-            )
+            into(input=image, output=target, name=name.capitalize(), identifier=identifier, con=con,
```
Comment (gwaybio) [inline cytominer_database/ingest.py]: this prob needs some linting?

File: cytominer_database/ingest.py [hunk 2]
```diff
@@ -168,9 +167,5 @@ def seed(source, target, config_path, skip_image_prefix=True):
         for compartment in compartments:
             name, _ = os.path.splitext(os.path.basename(compartment))
 
-            into(
-                input=compartment,
-                output=target,
-                name=name.capitalize(),
-                identifier=identifier,
-            )
+            into(input=compartment, output=target, name=name.capitalize(), identifier=identifier, con=con)
```
Comment (gwaybio) [inline cytominer_database/ingest.py]: this too?

**Review by gwaybio — APPROVED** [review]

File: cytominer_database/ingest.py (hunk 1 above)
Comment (d33bs) [reply cytominer_database/ingest.py]: Thanks, the formatting changed and we gained a parameter here. I'll try to keep the existing formatting in another change.

File: cytominer_database/ingest.py (hunk 2 above)
Comment (d33bs) [reply cytominer_database/ingest.py]: Thanks - see [comment](https://github.com/cytomining/cytominer-database/pull/135#discussion_r1795994038).

**Comment by d33bs** [comment]
Thanks @gwaybio ! Merging this in after making some formatting changes as you mentioned.

**Pull request merged**